pkg/provider/resource_proejct.go

```diff
@@ -0,0 +1,231 @@
+package provider
```

Comment:
There is a typo in the filename, must be `resource_project.go`, it is now `resource_proejct.go`